.travis.yml

```diff
@@ -1,28 +1,42 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
 
 sudo: false
 
 cache:
   directories:
     - node_modules
 
+env:
+  - EMBER_TRY_SCENARIO=ember-1.13
+  - EMBER_TRY_SCENARIO=ember-lts-2.4
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-beta
+  - EMBER_TRY_SCENARIO=ember-canary
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-canary
+
 before_install:
   - npm config set spin false
   - npm install -g bower
   - bower --version
   - npm install phantomjs-prebuilt
-  - phantomjs --version
+  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
 
 install:
   - npm install
   - bower install
 
 script:
-  - npm run-script test
-  - npm run-script jshint
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - npm run mocha
```

Comment:
this is failing because there is no `mocha` script defined. what's the reason for using `mocha` if all you do is check JSHint compliance?